README.md

```diff
@@ -74,4 +77,11 @@ we refer to collectively as "extensions") in the
 [Kong Hub](https://docs.konghq.com/hub), with documentation hosted
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
+## Contributing 
+We :heart: contributions from everyone! We welcome [bug reports](https://github.com/Kong/docs.konghq.com/issues/), [bug fixes](https://github.com/Kong/docs.konghq.com/pulls) and feedback on the library is always appreciated. Look at the [Contributor Guidelines](CONTRIBUTING.md) for more information.
```

Comment:
Please clarify that while github.com/kong/docs.konghq.com is the right place for bug reports and contributions (not just bug fixes) for the Kong docs site, our other repo https://github.com/Kong/kong is where folks should go for the Kong software itself - linking directly to https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-seek-for-help is a good idea.